data/cameras.xsd

```diff
@@ -514,11 +514,13 @@
   </xs:simpleType>
   <xs:simpleType name="CameraTypeSupportedType">
     <xs:restriction base="xs:string">
-      <xs:pattern value="(no|no-samples)"/>
+      <xs:pattern value="(no|no-samples|unknown|unknown-no-samples)"/>
```

Comment:
Maybe this should be two separate fields (`unsupported-no-samples`?),
but i suppose that can be done later.